bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx127-na.html

```diff
@@ -0,0 +1,112 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_127_na') }}
+{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-lg">
+    <h1 class="wnp-main-title">Join the community</h1>
```

Comment:
**Issue:** (non-blocking): Both the main title and the CTA say the same thing "Join the community", with no other information on what community you're actually joining. Reading the three blurbs below helps me to kinda piece together what it's about, but it feels like the page is missing a simple tag line explainer on what Connect is? The three blurbs should (imho) be extra information, and not a join-the-dots exercise?

I've added this as non-blocking as I'm sure copy has been approved already. But when seeing the page for the first time, it felt a bit too vague.

Reply:
I've asked for some alternatives, either a different headline or CTA, maybe both.